### PR TITLE
[SAM] Run SAM on AHUB

### PR DESCRIPTION
**Pull request by cgbahk**
This commit adds SAM configuration scripts. The content locations and
definition defined by AHUB.

ONE-DCO-1.0-Signed-off-by: Cheongyo Bahk <cg.bahk@gmail.com>

---

With this, we can run SAM analysis with SVACE

Tested in draft #6851

See https://github.com/Samsung/ONE/pull/6851#issuecomment-844701787 for detail

### CODE REVIEW

File: .ahub/sam/exclude.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/ONE/runtime/onert/frontend/tflite/src/tflite_schema_generated.h
+/ONE/runtime/nnapi-header/include/NeuralNetworks.h
+/ONE/runtime/nnapi-header/include/NeuralNetworksExtensions.h
+/ONE/runtime/libs/nnapi
+/ONE/runtime/libs/profiling
+/ONE/runtime/libs/tflite/port
+/ONE/runtime/3rdparty
+/ONE/compute
+/ONE/runtime/contrib
+/ONE/externals
```
Comment (cgbahk) [inline .ahub/sam/exclude.txt:15]: Note
- These are merged exclude from previous configuration for compiler and runtime.
- I really hope to add comment for each line, but not sure it is possible. Will do that in separate PR if possible.

File: .ahub/sam/advanced.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[SamPolicy]
+preset=CCD_FOR_OO
```
Comment (cgbahk) [inline .ahub/sam/advanced.cfg:2]: To use CCD instead of MCD

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Comment by cgbahk** [comment]
> With this, we can run SAM analysis with SVACE

I've missed this important notice :) Added now.

**Review by hseok-oh — APPROVED** [review]

**Comment by cgbahk** [comment]
Let's check whether this was effective on main branch
- compiler: https://analysishub.sec.samsung.net/service/analyses/539859
- runtime: https://analysishub.sec.samsung.net/service/analyses/539861

**Comment by cgbahk** [comment]
Both looks okay! :rocket:
